exporters/prometheus/config.go

```diff
@@ -132,7 +132,7 @@ func WithoutScopeInfo() Option {
 // have special behavior based on their name.
 func WithNamespace(ns string) Option {
 	return optionFunc(func(cfg config) config {
-		if model.NameValidationScheme != model.UTF8Validation {
+		if model.NameValidationScheme != model.UTF8Validation { // nolint:staticcheck // We need this check to keep supporting the legacy scheme.
```

Comment:
I think we can also give a notification via `global.Warn` so that users also can get the deprecation warning at runtime. See https://github.com/open-telemetry/opentelemetry-go/blob/66cb30e7890f7252ce93c70fd57a08ba7fadf1c2/sdk/log/record.go#L26-L28 for inspiration.

Reply:
Sure, that looks like a good idea.